unittests/data/BackendCountersTests.cpp

```diff
@@ -0,0 +1,186 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <data/BackendCounters.h>
+
+#include <boost/json/parse.hpp>
+#include <boost/json/serialize.hpp>
+#include <gtest/gtest.h>
+
+using namespace data;
+
+class BackendCountersTest : public ::testing::Test
+{
+protected:
+    static boost::json::object
+    emptyReport()
+    {
+        return boost::json::parse(R"(
+        {
+            "too_busy": 0,
+            "write_sync": 0,
+            "write_sync_retry": 0,
+            "write_async_pending": 0,
+            "write_async_completed": 0,
+            "write_async_retry": 0,
+            "write_async_error": 0,
+            "read_async_pending": 0,
+            "read_async_completed": 0,
+            "read_async_retry": 0,
+            "read_async_error": 0
+        }
+    )")
```

Comment:
Something odd happened to the formatting here

Reply:
I think the closing paren is misaligned, maybe that's why `.as_object()` is also moved.. idk